Bugsnag/include/Bugsnag/BugsnagStackframe.h

```diff
@@ -58,11 +64,18 @@
  */
 @property BOOL isLr;
 
+/**
+ * The type of the stack frame, if it differs from that of the containing error or event.
+ */
+@property(nullable) BugsnagStackframeType type;
```

Comment:
I think we'd want to use `BSGErrorType` rather than a new type as the possible enum values will always be the same as the error type.

Reply:
Unfortunately `BSGErrorType` was implemented as an integer-based enum and so cannot represent a "none" value which is a requirement for this new functionality.

I could change `BSGErrorType` to be a string-based enum but that seemed more disruptive.